File: WEKEE.BE/Product.Domain/Dto/UnitCardDtos.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Product.Domain.Dto
{
    public class UnitCardDtos
    {
        public ProductCardDtos productCardDtos { get; set; }
        public List<FeatureProductCardDtos> featureProductCardDtos { get; set; }
        public List<ImageProductCardDtos> imageProductCardDtos { get; set; }
        public List<HighlightProductCardDtos> highlightProductCardDtos { get; set; }
    }
}

File: WEKEE.FE/UI.HOME/src/scenes/Main/HDetail/dtos/unitCardDtos.ts
import { FeatureProductCardDtos } from "./featureProductCardDtos";
import { HighlightProductCardDtos } from "./highlightProductCardDtos";
import { ImageProductCardDtos } from "./imageProductCardDtos";
import { ProductCardDtos } from "./productCardDtos";

export interface UnitCardDtos {
  productCardDtos: ProductCardDtos;
  featureProductCardDtos: FeatureProductCardDtos[];
  imageProductCardDtos: ImageProductCardDtos[];
  highlightProductCardDtos: HighlightProductCardDtos[];
}
